messages/pt.json

```diff
@@ -1,9 +1,12 @@
 {
   "store/list-price.default": "Preço original: {listPriceValue}",
-  "store/savings.default": "Preço de venda: {newPriceValue}",
+  "store/selling-price.default": "Preço de venda: {sellingPriceValue}",
+  "store/savings.default": "Economize {savingsValue} ({savingsPercentage})",
   "store/installments.default": "ou {installmentsNumber}x de {installmentValue}{hasInterest, select, true {, {interestRate} de juros} false { sem juros}}",
   "admin/list-price.description": "Valores disponiveis para interpolação: '{listPriceValue}'",
   "admin/list-price.title": "List Price",
+  "admin/selling-price.description": "Valores disponiveis para interpolação: '{sellingPriceValue}'",
+  "admin/selling-price.title": "Selling Price",
```

Comment:
English?

Reply:
I am going to fix all these translation problems in the next PR, but thanks for commenting!